packages/spatial/src/transform/components/EntityTree.tsx

```diff
@@ -423,7 +423,7 @@ export function useAncestorWithComponents(
       unmounted = true
```

Comment:
I also wonder why this memory wasn't cleaning. Maybe there is something with StartReactor that we need to clean up. @HexaField @MichaelEstes